org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -108,6 +110,7 @@ function translatorSchemaToDeviceSchema(translatorSchema) {
 var deviceId;
 var deviceType = 'thermostats';
 var winkHelper;
+var accessToken;
```

Comment:
looks like this is only used by the common methods which means it can go away since the hub will have this info